src/ConfigurationTrait.php

```diff
@@ -83,11 +83,23 @@ public function getConfig($forceRefresh = false)
         $seedsPath = $this->getOperationsPath($this->input(), 'Seeds');
         $plugin = $this->getPlugin($this->input());
 
-        if (Configure::read('debug') && !is_dir($migrationsPath)) {
+        if (!is_dir($migrationsPath)) {
+            if (!Configure::read('debug')) {
+                throw new \RuntimeException(sprintf(
+                    'Migrations path `%s` does not exist and cannot be created because `debug` is disabled.',
```

Comment:
Thats not the reason imo
Better to output that path needs to exist 
No one should enable debug on prod

Reply:
Indeed it will never happen in prod because when code reaches prod the Migrations folder will exist. The problem happens when somebody disables debug in local environment and tries to create a migration

Reply:
I see :)

Reply:
@dereuromark this check works great if you are only using migrations but the seeds directory must exist now because of the check below to make sure the seeds directory exists. L96-L102